docs/graphql/core/api-reference/schema-metadata-api/table-view.rst

```diff
@@ -182,6 +224,11 @@ Table Config
      - Required
      - Schema
      - Description
+   * - identifier
```

Comment:
The name “identifier” here strikes me as very vague and a little confusing. We already use the phrase “custom fields” for the field names, why not call this a “custom name”?

Reply:
I have changed `identifier` to `custom name`